package.json

```diff
@@ -36,10 +36,12 @@
     "axios-retry": "^3.2.5",
     "commander": "^9.3.0",
     "fs-extra": "^9.1.0",
-    "graphql": "^16.3.0",
+    "graphql": "^14.2.1",
```

Comment:
why downgrade? we need to stay on `^16.3.0` for feeds-sdk.

Reply:
👍 `14.2.1` is the version used in canonical-reports. Will check if there are changes needed if I upgrade